website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -21,7 +21,7 @@ A secure Consul cluster should include the following:
 Before deploying your service, you will need to deploy the [ACL controller](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/acl-controller) so that it can provision the necessary tokens
 for tasks on the service mesh. To learn more about the ACL Controller, please see [Automatic ACL Token Provisioning](/docs/ecs/architecture#automatic-acl-token-provisioning).
 
-To deploy the controller, you will first need store an ACL token with `acl:write` privileges
+To deploy the controller, you will first need to store an ACL token with `acl:write` privileges
```

Comment:
Incidentally, we also need the controller's token to have `operator:write` now that it creates partitions and namespaces

Reply:
Good catch!